modules/ROOT/pages/_partials/data-sync/server-mqtt-pubsub.adoc

```diff
@@ -10,12 +10,13 @@ The xref:realtime-updates-{context}[] section describes how to set up the defaul
 
 == Using the MQTT PubSub mechanism
 
+// TODO: check link
```

Comment:
```suggestion

```

Link tested and works - it opened this url in a browser: https://www.npmjs.com/package/@aerogear/graphql-mqtt-subscriptions

Reply:
there are some links that use context attribute, but context isn't set, still TODO